### ISSUE TITLE
Incompatibility with a new 

### ISSUE THREAD

**Issue by LMSC-NTappy** [hyperspy/hyperspy#3056]
#### Describe the bug
Cannot load data with the new version of pint (released three days ago)

#### To Reproduce
```python
import hyperspy.api as hs
s = hs.load('data.hspy')
### Wild error appears
```

#### Expected behavior
pint.unit has disappeared in the new pint release. The following line causes the bug
https://github.com/hyperspy/hyperspy/blob/6a68fc4899409ea010b13c67721abce6a507022a/hyperspy/misc/axis_tools.py#L20

Fixing it should be as simple as replacing with
```python
from pint import Unit
```

But I haven't tested it

#### Python environement:
 - HyperSpy version: 1.7.2
 - Python version: 3.10
 - Pint 0.20.1
 
#### Additional context
Sorry for not giving more details, I was facing the problem when a colleague was installing hyperspy with me over zoom

**Comment by hakonanes** [comment]
Yeah, this will be fixed in #3052. As a temporary measure, you can downgrade `pint <= 0.19`.

**Comment by LMSC-NTappy** [comment]
Thanks for your reply. Yes this is exactly what we did :).

Sorry for reposting I didn't see it was already reported. 

Closing this now
